satviz-consumer/src/main/java/edu/kit/satviz/consumer/gui/config/GeneralConfigController.java

```diff
@@ -227,9 +227,9 @@ protected void setDefaultValues() {
 
     windowSizeSpinner.getValueFactory().setValue(ConsumerConfig.DEFAULT_WINDOW_SIZE);
 
-    coldColorColorPicker.setValue(GuiUtils.intToColor(HeatmapColors.DEFAULT_FROM_COLOR));
+    coldColorColorPicker.setValue(HeatmapColors.DEFAULT_HOT_COLOR);
```

Comment:
Siehe vorheriger Kommentar
```suggestion
    coldColorColorPicker.setValue(HeatmapColors.DEFAULT_COLD_COLOR);
```